src/Acuminator/Acuminator.Tests/Verification/VerificationHelper.cs

```diff
@@ -183,5 +195,49 @@ public static async Task<IEnumerable<Diagnostic>> GetCompilerDiagnosticsAsync(Do
 			var semanticModel = await document.GetSemanticModelAsync().ConfigureAwait(false);
 			return semanticModel.GetDiagnostics();
 		}
+
+		/// <summary>
+		/// Compile internal source code into memory array 
+		/// </summary>
+		/// <param name="sourceCodes">Source codes of library</param>
+		/// <returns>Byte array from compiled binary dll assembly</returns>
+		private static byte[] BuildAssemblyFromSources(string[] sourceCodes)
+		{
+			string assemblyName = Path.GetRandomFileName();
+
+			CSharpCompilation compilation = CSharpCompilation.Create(
+					assemblyName,
+					syntaxTrees: sourceCodes.Select(code => CSharpSyntaxTree.ParseText(text: code)),
+					references: new MetadataReference[] { CorlibReference,
+														  SystemCoreReference,
+														  CSharpSymbolsReference,
+														  CodeAnalysisReference,
+														  PXDataReference,
+														  PXCommonReference },
+					options: new CSharpCompilationOptions(OutputKind.DynamicallyLinkedLibrary));
+
+			// Emit the image of this assembly 
+			byte[] image = null;
+			using (var ms = new MemoryStream())
+			{
+				var emitResult = compilation.Emit(ms);
+				if (!emitResult.Success)
```

Comment:
Control flow operators must be separated with an empty line, according to our code style.